15/umbraco-cms/reference/umbraco-flavored-markdown.md

```diff
@@ -1,5 +1,9 @@
 # Umbraco Flavored Markdown
 
+{% hint style="info" %}
+This article also addresses the Label Property Configuration. If you were looking for that content, you're in the right place.
```

Comment:
In other places where we've done something similar (like the [Content Picker article](https://docs.umbraco.com/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/content-picker)), we ask a question.
So, maybe for consistency we do that here as well?

"Are you looking for Label Property Configuration?
"Umbraco Flavoured Markdown replaces Label Property Configuration in Umbraco 14+."